src/components/movement-sensor/client.test.ts

```diff
@@ -8,11 +8,33 @@ import { MovementSensorClient } from './client';
 
 let sensor: MovementSensorClient;
 
+interface Val {
+  toJavaScript(): string;
+}
+
+const mapVals: Val = {
```

Comment:
```suggestion
const mapVals: = {
```

Reply:
I'm realizing that we don't need a type here. We could probably just directly create mapVals